Investigate why running samples fails on windows

For some unknown reason test_samples.py fails when running on [windows](https://dev.azure.com/zivid-devops/zivid-python/_build/results?buildId=4477&view=logs&j=6ba2a0ce-315a-5ef0-a56f-83f4136474ba&t=4e8a91be-3506-5d6f-1c74-6bbe51ec615f&l=276)

This should be investigated

Tried this on my windows PC. It produces 3 errors, of which all seem reasonable:
```
FAILED test/test_camera.py::test_illegal_init - RuntimeError: Unsupported type for argument internal camera: this should fail, type: <class 'str'>.
FAILED test/test_point_cloud.py::test_illegal_init - ValueError: Unsupported type for argument internal_point_cloud. Got <class 'str'>, expected <class 'pybind11_builtins.pybind11_type'>
FAILED test/test_zivid_module_content.py::test_version - AssertionError: assert '0.9.1.1.7.0' == '1.0.0.1.8.1'
```
The first two failures "should fail". However, shouldn't a wanted failure be logged as success?

I tested this on master, but against `python -m pip install zivid`. Why would `zivid.__version__` be 0.9.1.1.7.0?

--- a/test/conftest.py
+++ b/test/conftest.py
@@ -114,13 +114,8 @@ class Cd:
 
 @pytest.helpers.register
 def run_sample(name, working_directory=None):
-    current_working_directory = Path(os.getcwd()).resolve()
     sample = (
-        current_working_directory
-        / ".."
-        / ".."
-        / "samples"
-        / "sample_{name}.py".format(name=name)
+        Path(__file__) / ".." / ".." / "samples" / "sample_{name}.py".format(name=name)
     ).resolve()
 
     if working_directory is not None:

--- a/test/test_samples.py
+++ b/test/test_samples.py
@@ -1,21 +1,31 @@
-import platform
-
 import pytest
 
 
-@pytest.mark.skipif(
-    platform.system() == "Windows",
-    reason=r"python: can't open file 'D:\a\samples\sample_capture_from_file.py': [Errno 2] No such file or directory",
-)
 def test_capture_from_file(sample_data_file):
     pytest.helpers.run_sample(
         name="capture_from_file", working_directory=sample_data_file.parent
     )
 
 
-@pytest.mark.skipif(
-    platform.system() == "Windows",
-    reason=r"python: can't open file 'D:\a\samples\sample_print_version_info.py': [Errno 2] No such file or directory",
-)
 def test_print_version_info():
     pytest.helpers.run_sample(name="print_version_info")
+
+
+@pytest.mark.physical_camera
+def test_capture():
+    pytest.helpers.run_sample(name="capture")
+
+
+@pytest.mark.physical_camera
+def test_capture_2d():
+    pytest.helpers.run_sample(name="capture_2d")
+
+
+@pytest.mark.physical_camera
+def test_capture_assistant():
+    pytest.helpers.run_sample(name="capture_assistant")
+
+
+@pytest.mark.physical_camera
+def test_capture_hdr():
+    pytest.helpers.run_sample(name="capture_hdr")

